platform/legacy/__tests__/LegacyKbnServer.test.ts

```diff
@@ -0,0 +1,28 @@
+import { LegacyKbnServer } from '..';
+
+it('correctly returns `newPlatformProxyListener`.', () => {
```

Comment:
nitpick: `it` -> `test`?

Reply:
(I'm not sure here, as I don't know if we are consistent either way, so feel free to skip)

Reply:
ha-ha, right, it should definitely be `test` - I believe we are pretty consistent on this in the new platform code (I see only 3 more tests that use `it` and that was me who did that :), so I'll change them to `test` too).